src/sentry/tasks/post_process.py

```diff
@@ -1218,6 +1218,27 @@ def plugin_post_process_group(plugin_slug, event, **kwargs):
     )
 
 
+def feedback_decorator(func):
```

Comment:
nit: I'd give it more explicit name, like _"skip_unless_feedback_decorator"_ or something like that

Reply:
totally. let me fix that.